### ISSUE TITLE
[Feature request] multiple downloads

### ISSUE THREAD

**Issue by Dav-Renz** [ihabunek/twitch-dl#50]
Would it be possible to list multiple streams in one command, and download them one after another, merging them before starting with the next one? 

Also is it possible to check the free disk space beforehand, since you need double the space of the actual download while merging.

Thanks in advance, and great job you're doing here

**Comment by ihabunek** [comment]
Possible, needs work. Checking space beforehand is tricky because I don't know the size of the VODs before they're downloaded.

**Comment by ihabunek** [comment]
This has been available since [v2.0.0](https://github.com/ihabunek/twitch-dl/releases/tag/2.0.0). 

```
twitch-dl download <id1> <id2> <id3> ...
```

Could you open another issue for checking free space please?
